content/tutorials/portenta-h7/por-openmv-gs/content.md

```diff
@@ -0,0 +1,211 @@
+when it will reset with the `maxWidth` variable.
+
+```py
+# maximum and minimum pulse-width, corresponds to maximum and minimum brightness
+maxWidth = 5000
+minWidth = 0
+
+# How much to change the pulse-width by each step
+step = 500
+# Set starting value
+curWidth = minWidth
+```
+
+### 5. Creating the Main Loop in the Script
+
+Putting our code inside a `while True:` function will make the code run in a loop.
+
+We use the `channel1.pulse_width()` function to set the pulse width of the PWM channel, changing the intensity of the LED. Using `pyb.delay()` we can make the script stop running for the determined amount of time, which is 500 milliseconds in this example. At the end, we check if the value has reached our defined end, we print a message to the serial terminal and reset the `curWidth` variable.
+
+```py
+while True:
+
+  channel1.pulse_width(curWidth)
+
+  # this determines how often we change the pulse-width.
+  pyb.delay(500)
+
+  curWidth = curWidth + step
+
+  if curWidth > maxWidth:
+    print("Max width reached!")
+    curWidth = minWidth
+```
+
+### 6. Uploading the Script
+
+Heres the complete sketch for generating a PWM on the I/O pin 5. Connect your board to OpenMV and upload the following sketch.  
+
+```py
+import pyb # Import module for board related functions
+import time # Import module for board related functions
+from pyb import Pin, Timer # Import module for board related functions
+
+pin1 = Pin("PC6", Pin.OUT_PP, Pin.PULL_NONE)
+
+timer1 = Timer(3, freq=1000) # Frequency in Hz
+
+channel1 = timer1.channel(1, Timer.PWM, pin=pin1, pulse_width=0)
+
+# Maximum and minimum pulse-width, corresponds to maximum and minimum brightness
+maxWidth = 5000
+minWidth = 0
+
+# How much to change the pulse-width by each step
+step = 500
+# Set starting value
+curWidth = minWidth
+
+while True:
+  channel1.pulse_width(curWidth)
+
+  # How often the pulse-width is changed
+  pyb.delay(500)
+  curWidth = curWidth + step
+
+  if curWidth > maxWidth:
+    print("Max width reached!")
+    curWidth = minWidth
+```
+
+To upload the code to your board in OpenMV, press the green play button in the lower left corner.
+
+![Press the green play button to upload the script](assets/por_openmv_board_connected.png)
+
+If the wiring is correct the LED light should start incrementally getting brighter before resetting and starting over. Printing a message when the limit has been reached. If everything went correctly you should get the same output in the serial terminal as seen below.
+
+![Screenshot of openmv showing what will print in the serial terminal](assets/por_openmv_ide_result.png)
+
+## Conclusion
+
+In this tutorial, you learned how to use the OpenMV IDE to develop MicroPython scripts that run on the Portenta board. You learned how to use and configure I/O pins and PWM on the Portenta. Lastly, you learned how to use these to control a LED light.
+
+
+### Next Steps
```

Comment:
@BenjaminDannegard This section is rather meant to describe next steps based on what they learned here. What else could they explore that is related to this? I'd avoid referring to the other tutorials unless they are an actual sequel to this one.

Reply:
Alright I will change that section